tests/scripts/run-throughput-perf.py

```diff
@@ -0,0 +1,361 @@
+#!/usr/bin/env python
```

Comment:
Currently, all of our lab machines have both Python 2.7 and Python 3.5 installed.  2.7 is for the CoreCLR build while Python 3.5 is for the Benchview tools as @adiaaida mentioned above.